README.md

```diff
@@ -1,14 +1,35 @@
 # HyP3-lib
 
-## Common library for HyP3 plugins
+## CURRENTLY UNDER CONSTRUCTION!!!
+HyP3-lib is currently undergoing a major redesign and many utilities that are no longer used by ASF have been removed between version 2 and version 3. In addition, we plan to migrate some remaining functionality to other projects and to significantly refactor what remains. For this reason, **if you are using HyP3-lib we strongly recommend pinning to major versions at this time**. We further recommend not using this library in production until its version 4 release.
```

Comment:
```suggestion
> [!WARNING]
> HyP3-lib is currently undergoing a major redesign and many utilities that are no longer used by existing HyP3 plugins have been removed between version 2 and version 3. In addition, we plan to migrate some remaining functionality to other projects and to significantly refactor what remains. For this reason, **if you are using HyP3-lib we strongly recommend pinning to major versions at this time**. We further recommend not using this library in production until its version 4 release.
```
I think using an alert here would pop better and be more appropriate
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts